test/unit/Ramm/setup.js

```diff
@@ -17,14 +17,14 @@ async function getState(ramm) {
   return { nxmA, nxmB, eth, budget, ratchetSpeed, timestamp };
 }
 
-async function setup() {
+async function setup(mockPool = 'RammMockPool') {
```

Comment:
not sure if I'm super happy with this but not sure how else to do it w/o copy pasting the whole setup function.

happy to hear suggestions how to do this better.

Reply:
Fixed using `setCode`